### PR TITLE
Update contingency list for lines and transformers

### PR DESCRIPTION
**Pull request by dbraquart**

### CODE REVIEW

File: src/translations/en.json [hunk 1]
```diff
@@ -149,6 +141,8 @@
     "nominalVoltage2": "Nominal Voltage 2",
     "nominalVoltage3": "Nominal Voltage 3",
     "cannotPasteTextAsNominalVoltage": "The clipboard text cannot be set as a positive number :",
+    "cannotRetrieveContingencyList": "Could not retrieve contingency list :",
+    "cannotRetrievefilter": "Could not retrieve filter :",
```
Comment (AbdelHedhili) [inline src/translations/en.json]: in English there is never a space before the ":" character and always a space after
"cannotRetrieveContingencyList": "Could not retrieve contingency list: ",
"cannotRetrievefilter": "Could not retrieve filter: ",

File: src/translations/fr.json [hunk 2]
```diff
@@ -148,6 +140,8 @@
     "nominalVoltage2": "Tension nominale 2",
     "nominalVoltage3": "Tension nominale 3",
     "cannotPasteTextAsNominalVoltage": "Le texte copié ne peut pas être utilisé pour un nombre positif :",
+    "cannotRetrieveContingencyList": "Erreur d'accès à la liste d'aléas :",
+    "cannotRetrievefilter": "Erreur d'accès au filtre :",
```
Comment (AbdelHedhili) [inline src/translations/fr.json]: space after ":" 
"cannotRetrieveContingencyList": "Erreur d'accès à la liste d'aléas : ",
"cannotRetrievefilter": "Erreur d'accès au filtre : ",

File: src/translations/en.json (hunk 1 above)
Comment (dbraquart) [reply src/translations/en.json]: ok, I will change

File: src/translations/fr.json (hunk 2 above)
Comment (dbraquart) [reply src/translations/fr.json]: ok, I will change. didnt know it was different between fr et en (space before / no space before).

File: src/utils/equipment-types.js [hunk 3]
```diff
@@ -1,17 +1,307 @@
...
+            },
+            nominalVoltage: {
+                name: 'nominalVoltage',
+                type: filteredTypes.range,
+            },
+        },
+    },
+};
+
+export const contingencyListEquipmentDefinition = {
```
Comment (FranckLecuyer) [inline src/utils/equipment-types.js:166]: Maybe only one equipment definition data structure for contingency and filter ?

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]

File: src/utils/equipment-types.js (hunk 3 above)
Comment (dbraquart) [reply src/utils/equipment-types.js:166]: Well its not easy to have a common data: the fields names are the json attribute names, and they are not the same in the filter/alea data model.
I kept 2 lists, but I made them lighter, easier to read

File: src/translations/fr.json [hunk 4]
```diff
@@ -161,7 +153,9 @@
     "nominalVoltage1": "Tension nominale 1",
     "nominalVoltage2": "Tension nominale 2",
     "nominalVoltage3": "Tension nominale 3",
-    "cannotPasteTextAsNominalVoltage": "Le texte copié ne peut pas être utilisé pour un nombre positif :",
+    "cannotPasteTextAsNominalVoltage": "Le texte copié ne peut pas être utilisé pour un nombre positif : ",
+    "cannotRetrieveContingencyList": "Erreur d'accès à la liste d'aléas : ",
+    "cannotRetrievefilter": "Erreur d'accès au filtre :",
```
Comment (AbdelHedhili) [inline src/translations/fr.json]: "cannotRetrievefilter": "Erreur d'accès au filtre : " 
space after ":" (sorry !)

**Review by AbdelHedhili — APPROVED** [review]

File: src/translations/fr.json (hunk 4 above)
Comment (dbraquart) [reply src/translations/fr.json]: yes, I changed the line before. I will update the last one.

**Review by FranckLecuyer — APPROVED** [review]

**Pull request merged**